server/sanitize/index.js

```diff
@@ -1,19 +1,22 @@
+/** @format */
+
 /**
  * Encode single characters with backslashes.
+ *
  * @param {string} charStr single-character string.
  * @return {string} backslash escaped character.
- *
  * @copyright (c) 2013, GoInstant Inc., a salesforce.com company.
  * @license See CREDITS.md.
  * @see https://github.com/goinstant/secure-filters/blob/master/lib/secure-filters.js
- *
  * @private
- **/
+ */
+
 function jsSlashEncoder( charStr ) {
 	var code = charStr.charCodeAt( 0 ),
 		hex = code.toString( 16 ).toUpperCase();
 
-	if ( code < 0x80 ) { // ASCII
+	if ( code < 0x80 ) {
+		// ASCII
```

Comment:
I guess this comment should go before if statement now. It isn't prettier's fault though :)